bin/configs/server.conf.example

```diff
@@ -4,7 +4,9 @@ server.socket_host='0.0.0.0'
 server.socket_port=8080
 server.thread_pool=20
 tools.sessions.on=True
+tools.sessions.clean_freq=2
```

Comment:
@grondinjc I will add a comment.
The point is that once a session expire, it's not removed from the sessions cache right away. So, when we're looking at the sessions to check if the username is already in use, the expired session are still there. The clean_freq=2 means "every 2 minutes, clear the expired session"
So, if your session expire it'll take a maximum of 2 minutes before you can log with the same user again.